doc/src/modules/polys/domainsintro.rst

```diff
@@ -48,9 +48,9 @@ representing the implementation of those coefficients::
   Poly(x**2 + 1/2*x, x, domain='QQ')
   >>> p.domain
   QQ
-  >>> p.rep
-  DMP([1, 1/2, 0], QQ)
-  >>> p.rep.rep
+  >>> p.rep  # doctest: +SKIP
+  DMP_Python([1, 1/2, 0], QQ)
+  >>> p.rep.rep  # doctest: +SKIP
```

Comment:
Parts of this doc page will need to be rewritten at some point but I don't want to do it now because I anticipate more changes that would make anything I say now become out of date quite quickly. For now I have disabled some doctests that would fail if ground types is set to flint when running the doctests.